src/test/java/lotto/oop/domain/LottoIssueTest.java

```diff
@@ -21,4 +24,13 @@ public void publishLootListTest() {
         lotto.publishLottoList();
         assertThat(lotto.getResultList().size()).isEqualTo(14);
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"46 47 48 49 50 51"})
+    void checkNumberTest(String numbers) {
+        String[] numList = numbers.split("");
```

Comment:
```suggestion
        String[] numList = numbers.split(" ");
```
의도하지 않은 다른 이유로 에러가 발생하고 있을 것 같아요.